assets/sass/_footer.scss

```diff
@@ -17,7 +17,7 @@
       font-size: $font-size-normal;
     }
 
-    @media screen and (max-width: $width-phone) {
+    @include max-width($width-phone) {
```

Comment:
Expected item on line 20 to appear before line 11. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets